rollup.config.js

```diff
@@ -54,5 +54,15 @@ export default [
     },
     external: ["http", "https"],
     plugins: createPlugins()
+  },
+  {
+    input: "lib/entry-umd.ts",
+    output: {
+      format: "iife",
```

Comment:
a umd has to be an iife? this seems wrong to me, that would mean we'd also have to do this for all other libraries

Reply:
I guess you could see UMD a special kind of IIFE that's doing specific things to be compatible with AMD, CommonJS, etc.
The real issue is the fact that we can't load an UMD module via a `<script>` tag if require.js happens to be on that page as well, which is too bad because  that's precisely what the snippet does. 

For most libraries, a UMD build is best because if you do have Require.js on your page, you should use it to fetch and use all your other libraries (InstantSearch.js, search client, etc)

Reply:
As @eunjae-lee suggested, we could add the IIFE build instead of replacing the UMD one. Users who need it can import the IIFE build, others won't be impacted.